sources/academy/webscraping/puppeteer_playwright/page/interacting_with_a_page.md

```diff
@@ -36,15 +36,15 @@ Let's first focus on the first 3 steps listed above. By using `page.click()` and
 <TabItem value="Playwright" label="Playwright">
 
 ```js
-// Click the "I agree" button
+// Click the "Accept all" button
```

Comment:
We could also address https://github.com/apify/apify-docs/issues/418 when we're already at this 💡

Reply:
We could, but it is fine since the reader should reflect on it in academy tutorials. This fix addresses the consistency between the text and the examples.